src/content-helper/editor-sidebar/class-editor-sidebar.php

```diff
@@ -106,11 +106,11 @@ private function get_parsely_post_url( $post_id = null, bool $add_utm_params = t
 		/**
 		 * The post object.
 		 *
-		 * @var WP_Post $post
+		 * @var WP_Post|null $post
 		 */
 		$post = get_post( $post_id );
 
-		if ( ! Dashboard_Link::can_show_link( $post, $this->parsely ) ) {
```

Comment:
This fixes an unrelated test failure due to a type issue I've seen a couple of times [[1]](https://github.com/Parsely/wp-parsely/actions/runs/14390004182/job/40354625985) [[2]](https://github.com/Parsely/wp-parsely/actions/runs/14452501906/job/40528286468#step:12:595).